includes/embeds/class-amp-youtube-embed-handler.php

```diff
@@ -151,10 +245,59 @@ public function render( $args, $url ) {
 				'width'       => $this->args['width'],
 				'height'      => $this->args['height'],
 				'placeholder' => '',
+				'start'       => 0,
 			]
 		);
 
-		if ( empty( $args['video_id'] ) ) {
+		$attributes = array_merge(
+			[ 'data-videoid' => $args['video_id'] ],
+			wp_array_slice_assoc( $args, [ 'layout', 'width', 'height' ] )
+		);
```

Comment:
What about directly creating a new array instead of merging and slicing?

```suggestion
		$attributes = [
			'data-videoid' => $args['video_id'],
			'layout'       => $args['layout'],
			'width'        => $args['width'],
			'height'       => $args['height'],
		];
```